oxen-rust/src/server/src/services/versions.rs

```diff
@@ -25,11 +25,11 @@ pub fn versions() -> Scope {
             web::post().to(controllers::versions::chunks::complete),
         )
         .route(
-            "/{version_id}/chunks/download",
+            "/chunks/{resource:.*}/download",
```

Comment:
Let's keep the `/versions` endpoints just using `version_id` and use the `/file` endpoints when we don't know the version ID already.